### ISSUE TITLE
Safari-Erweiterung

### ISSUE THREAD

**Issue by Chris-topher-Allen** [stefanw/bibbot#239]
auf https://stefanw.github.io/bibbot/#safari lässt sich nicht laden. "Not found"

Im Appstore ist nur eine uralte Version eingestellt...

**Comment by BenutzerEinsZweiDrei** [comment]
Hier ist zumindest die letzte Version

https://github.com/stefanw/bibbot/releases/tag/v0.26.5

**Comment by Chris-topher-Allen** [comment]
Oh, Danke! - als ich das letzte Mal nachgesehen hatte, war die dort noch nicht eingestellt...?

**Comment by bogenschlag** [comment]
Von v0.26.6 gibt es offenbar (bislang) kein Release für Safari. Nur @stefanw weiß, warum. 🤷🏼‍♂️😌

**Comment by Chris-topher-Allen** [comment]
Ja, merkwürdig. Auch unter release ist diesmal nichts zu finden.

**Comment by stefanw** [comment]
Der Safari-Release muss leider gerade noch manuell erstellt werden und ist nicht in der automatischen release pipeline.
